scripts/crunner.py

```diff
@@ -215,9 +207,10 @@ def print_output(*args):
   spitfire.runtime.udn.set_accelerator(
     options.enable_c_accelerator, enable_test_mode=True)
 
-  compiler_args = Compiler.args_from_optparse(options)
-  compiler = Compiler(**compiler_args)
-  
+  compiler_args_from_optparse = (
+      spitfire.compiler.compiler.Compiler.args_from_optparse(options))
+  compiler = spitfire.compiler.compiler.Compiler(**compiler_args_from_optparse)
```

Comment:
compiler.compiler.compiler.Compiler.compile(compile)... lolwat

What if we do:

from spitfire.compiler import sptcompiler

sptcompiler.Compiler()

we can do the same for visitor, util, etc.